qcfractal/interface/client.py

```diff
@@ -107,15 +107,17 @@ def __init__(self,
                               f"the one of following commands (pip or conda):"
                               f"\n\t- pip install qcportal=={self.server_info['version']}"
                               f"\n\t- conda install -c conda-forge qcportal=={self.server_info['version']}"
+                              f"\n(Only MAJOR.MINOR versions are checked)"
                               )
             client_version = _version_list(__version__)
-            if not server_version_min_client <= client_version <= server_version_max_client:
+            if not server_version_min_client[:2] <= client_version[:2] <= server_version_max_client[:2]:
                 raise IOError(f"This Client of version {client_version} does not fall within the Server's allowed "
                               f"Client versions of [{server_version_min_client}, {server_version_max_client}] at "
```

Comment:
We need to change these lines to only show major/minor versions.

Reply:
I guess this is somewhat fixed by setting the parameters in the server. Still maybe do it for the client?